packages/SwingSet/src/liveslots/vatstore-usage.md

```diff
@@ -24,11 +24,11 @@ Liveslots maintains three counters to create the distinct vrefs that it transmit
 * `collectionID`: each collection gets the next ID
 * `promiseID`: each exported Promise and outbound-message `result` gets the next ID
```

Comment:
@warner Can you provide precise starting points, increment conditions, and formats for these counters? For example, would this be right?
```suggestion
* `collectionID`: an integer indicating the Collection ID of the next collection and incremented upon each allocation of a new collection. Starts at `1`, which is claimed by "[baggage](#baggage)".
* `promiseID`: an integer indicating the ID for the next exported promise (including a promise implicitly created for outbound-message results). Starts at 1.
```